packages/repco-webcomponent/src/post-card/post-card-body.ts

```diff
@@ -0,0 +1,27 @@
+import {css, html, LitElement} from 'lit';
+import {customElement, property} from 'lit/decorators.js';
+import {unsafeHTML} from 'lit/directives/unsafe-html.js';
+
+@customElement('post-card-body')
```

Comment:
we should prefix all custom elements with `repco-` to not have conflicts. custom elements share a global namespace per page unfortunately.